tests/staticfiles_tests/test_storage.py

```diff
@@ -172,6 +172,7 @@ def test_post_processing(self):
         self.assertIn(os.path.join('cached', 'css', 'window.css'), stats['post_processed'])
         self.assertIn(os.path.join('cached', 'css', 'img', 'window.png'), stats['unmodified'])
         self.assertIn(os.path.join('test', 'nonascii.css'), stats['post_processed'])
+        self.assertIn(os.path.join('cached', 'css', 'ignored.css'), stats['post_processed'])
```

Comment:
@timgraham right, I've changed `test_path_ignored_completely` to test for regression with `chrome:`